### ISSUE TITLE
Pytorch Implementation? 

### ISSUE THREAD

**Issue by shreyaskamathkm** [gaudetcj/DeepQuaternionNetworks#5]
Hi, 

Do you have a pytorch implementation of this code?  It would be very helpful even if you have a base code?

Thank you.

**Comment by gaudetcj** [comment]
Sorry for long response time, but here is a PyTorch repo from another person researching quaternion networks

https://github.com/Orkis-Research/Pytorch-Quaternion-Neural-Networks

**Comment by EdoardoCicero** [comment]
Hi,
i know this is closed but since the repository of TParcollet that you posted doesn't have your quaternion batch normalization in PyTorch do you have any plan to implement it in that framework? I'm trying to do it but i'm new to pytorch and i'm having doubts about what i'm doing.

Thank you.

**Comment by shreyaskamathkm** [comment]
Hi,

Please take a look at  [this](https://github.com/Riccardo-Vecchi/Pytorch-Quaternion-Neural-Networks/blob/master/core/quaternion_layers.py). They have a quaternion batch norm implementation.

Thank you,
